spec/features/users/01_user_create_account_spec.rb

```diff
@@ -11,52 +11,55 @@
   # [ ] Form returns error when required fields are not entered correctly
   # [ ] Displays success message and redirects to user profile when form
   #     is submitted correctly
+  context "as a new user" do
```

Comment:
New user implies they recently joined. These specs are describing someone that has not yet joined. Maybe as a prospective user?